src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -8044,28 +8028,13 @@ components:
       required: true
       schema:
         type: string
-    Depth:
-      name: depth
-      in: query
-      description: Enumeration depth; -1 means fully expand; 0 means no expansion;
-        1..N expands that many subdirectories
-      schema:
-        type: integer
-        default: 0
     LoadTag:
       name: loadtag
       in: path
       description: a load tag
       required: true
       schema:
         type: string
-    AssetId:
```

Comment:
Would it be better to use this (for `removeDatasetAssetSpecifications`) rather than removing it from the schema?

Also I see that this says `An asset id` but this is actually the asset name.